CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 [Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-aws/compare/0.16.0...HEAD
 
+### Changed
+- update variable type `dimensions` in Cloudwatch probes get_metric_statistics and get_metric_data 
+
```

Comment:
Could this be updated slightly to be a bit more specific, along the lines of:

- `get_metric_statistics` and `get_metric_data` now take a `list[dict[str, Any]]` of `dimensions` rather than one `dimension_name` and `dimension_value` to allow for multiple dimension queries

Reply:
Also, there's already a ### Changed section on line 20:21, could you move this list entry there?

Reply:
I added next text in line 19:
- added new variable for `get_metric_statistics` and `get_metric_data` called `dimensions: list[dict[str, Any]]` to allow for multiple dimension queries